.github/workflows/ci.yaml

```diff
@@ -13,7 +13,6 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-        - '2.6'
```

Comment:
We should blast this out to the other plugins if we are dropping 2.6

Reply:
https://github.com/ManageIQ/manageiq-release/pull/259